enhancements/machine-api/azure-ultra-disks.md

```diff
@@ -48,7 +48,7 @@ Allow users running OCP clusters on Azure to leverage Ultra Disk Storage to achi
 ## Proposal
 
 The following requirements for integration will be necessary for adding Ultra Disks support:
-1. Required configuration for supporting Data Disk should be added to the `AzureMachineProviderSpec`. 
+1. Required configuration for supporting Data Disk should be added to the `AzureMachineProviderSpec`
```

Comment:
Why did we lose the period at the end here?

Reply:
Yes that's because I didn't have it on the follow up second point neither, so I decided to remove that from here as well.

Reply:
Looking at the rendered doc, it should definitely be on this one at least else you just randomly start a new sentence after the `AzureMachineProviderSpec`

Reply:
Oh yes, right, will add back.